src/Components/Events/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import SectionSubheader from '../SectionSubheader';
+import GoogleCalendar from './GoogleCalendar';
+import { Box, MainContainer, Content, Description } from './style';
+
+function Events() {
+
+  const renderCalendar = () => {
+    return(
+      <Box>
+        <Content>
+          <SectionSubheader title="Event Calendar"/>
+          <Description>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer malesuada massa nec porta interdum. 
```

Comment:
Not sure about the content yet, but you could mention the external links to the calendar in different timezones. Here's the text you can put:
```suggestion
          We host multiple open sessions to the community where members of the community can discuss concerns about the community, ask questions, discuss projects, etc. These sessions are open to anyone and are usually conducted via Zoom. These are audio-only calls, so no video is required.
          You can see our calendar in different timezones:
```

Then in front of ":" you could add HTML code to link each timezone to its calendar like: ([PST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles) || [EST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/New_York) || [GMT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=GMT) || [WAT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Africa/Lagos) || [IST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Asia/Colombo) || [AEDT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Australia/Sydney))

@shravani05

Reply:
@isabelcosta On it!